shared/actions/crypto.tsx

```diff
@@ -56,6 +57,16 @@ function* teamBuildingSaga() {
   yield* Saga.chainAction2(TeamBuildingGen.finishedTeamBuilding, filterForNs('crypto', onSetRecipients))
 }
 
+// more of a debounce to keep things simple
+let throttleSetInputCurrentAction: CryptoGen.SetInputPayload | undefined
```

Comment:
Originally the component would debounce calling the dispatch but we want to react to this immediately. We do want to handle the side effects of making the rpc call in a deferred manner so this belongs here